src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -64,8 +83,16 @@ class AccessControlAttribute : public AttributeAccessInterface, public EntryList
     CHIP_ERROR Write(const ConcreteDataAttributePath & aPath, AttributeValueDecoder & aDecoder) override;
 
 public:
-    void OnEntryChanged(const SubjectDescriptor * subjectDescriptor, FabricIndex fabric, size_t index, const Entry * entry,
-                        ChangeType changeType) override;
+    void OnEntryChanged(const SubjectDescriptor * subjectDescriptor, FabricIndex fabric, size_t index,
```

Comment:
Is there a really compelling reason to change the signature? Each refactor increases risks given the late landing of the feature.

Reply:
The signature didnt change, I just had to add namespaces to existing types to prevent conflicts.